aries-site/src/examples/components/datatable/DataTableVerticalListExample.js

```diff
@@ -0,0 +1,107 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import {
+  Box,
+  Button,
+  DataTable,
+  Heading,
+  Text,
+  ResponsiveContext,
+} from 'grommet';
+import ServerGroups from '../../../data/mockData/serverGroups.json';
+
+/* This function displays an arr's values in a vertical list within a table 
+   cell, hiding all values from view that exceed the given maximum list 
+   length until the "View More" button is clicked, at which point the 
+   button label changes to "View Less". */
+const ViewMore = (data, max) => {
+  const [viewMore, setViewMore] = useState(false);
+  return (
+    <Box>
+      {viewMore
+        ? data.map(item => <Text>{item}</Text>)
+        : data.slice(0, max).map(item => <Text>{item}</Text>)}
+
+      {data.length > max && (
+        <Button
+          alignSelf="start"
+          label={viewMore ? 'View less' : `View more (${data.length - max})`}
+          onClick={() => setViewMore(!viewMore)}
+        />
+      )}
+    </Box>
+  );
+};
+
+const data = ServerGroups.groups;
+const max = 3;
+const columns = [
+  {
+    property: 'name',
+    header: 'Name',
+    render: datum => <Text truncate>{datum.name}</Text>,
+  },
+  {
+    property: 'groupName',
+    header: 'Group Name',
+    render: datum => <Text truncate>{datum.groupName}</Text>,
+    sortable: true,
+  },
+  {
+    property: 'servers',
+    header: 'Servers',
+    sortable: false,
+    render: datum => ViewMore(datum.servers, max),
+  },
+  {
+    property: 'status',
```

Comment:
Realizing this late, but we've been going for the convention that Status column appears earlier in the datatable so it's more visible if something goes wrong. Can we move this to the first column? and render status icon how it is shown in this example (ignore that it's in last column, old example)? https://deploy-preview-2705--keen-mayer-a86c8b.netlify.app/components/datatable?q=data#sorting

Reply:
You can reference the code for that example as a how-to.